src/NServiceBus.Core/UnitOfWork/UnitOfWorkBehavior.cs

```diff
@@ -54,7 +55,7 @@ await popped.End()
             }
             catch (Exception exception)
             {
-                var trailingExceptions = await AppendEndExceptions(unitsOfWork, exception).ConfigureAwait(false);
+                var trailingExceptions = await AppendEndExceptions(unitsOfWork, exception, context.CancellationToken).ConfigureAwait(false);
```

Comment:
Not related to this PR but since we obsoleted the UoW with ERR do we really need this behavior anymore? cc @DavidBoike